openspec/changes/docs-15-code-review-validation-guardrails/proposal.md

```
@@ -0,0 +1,45 @@
+## Why
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)